src/System.Diagnostics.FileVersionInfo/tests/Assembly1.csproj

```diff
@@ -30,7 +30,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.Runtime">
-      <HintPath>..\..\packages\System.Runtime.4.0.20-beta-22412\lib\portable-wpa80+win80+net45+aspnetcore50\System.Runtime.dll</HintPath>
+      <HintPath>..\..\..\packages\System.Runtime.4.0.20-beta-22412\lib\portable-wpa80+win80+net45+aspnetcore50\System.Runtime.dll</HintPath>
```

Comment:
I believe this reference should be removed; this should be resolved from the packages.config file for this assembly. That is, unless it's not working for this assembly because it uses packages.Assembly1.config.